### PR TITLE
Added new scheme cartes_bancaires to enum

### PR DESCRIPTION
**Pull request by thyago-falconi-cko**
# Description of changes in PR

Added new scheme cartes_bancaires to enum

## Checklist

- [x] Added a changelog entry to the `changelog.md` file in either `abc_spec` or `nas_spec`
- [ ] Contacted the Tech Docs team about any corresponding guides that need to be updated for www.checkout.com/docs

**Contributing options**: Look at [our documentation](https://checkout.atlassian.net/wiki/spaces/PD/pages/2169506663/API+ref+publication+process) for options to contribute to the API reference.

### CODE REVIEW

**Comment by thyago-falconi-cko** [comment]
@chrisi-webster-cko can you please have a look at this when you have a minute? Many thanks, 👍🏽

File: nas_spec/changelog.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 | Date       | Description of change                                                                                                 |
 | ---------- | --------------------------------------------------------------------------------------------------------------------- |
-| 2022/03/22 | Fixed invalid format for `authentication_date` |
+| 2022/03/22 | Added new scheme `cartes_bancaires` to enum                                                                           |
```
Comment (ben-ahmady-cko) [inline nas_spec/changelog.md:5]: Which enum? :)

**Review by ben-ahmady-cko — DISMISSED** [review]
@thyago-falconi-cko If it's too messy to clarify the enum here and we already have a CHANGELOG entry indicating support for CB, don't worry about this too much

You will need someone to reapprove after if you make a new commit, just ping me again

File: nas_spec/changelog.md (hunk 1 above)
Comment (thyago-falconi-cko) [reply nas_spec/changelog.md]: hey @ben-ahmady-cko, thanks for your feedback. I wasn't sure how detailed the changelog needed to be. I've added a bit more detail on this commit: https://github.com/checkout/checkout-api-reference/pull/539/commits/9ea2eeff84a04efd8c1ca93b19141428864713ac
Can you please have a look? Many thanks.

**Review by smaranda-vladoiu-cko — APPROVED** [review]

File: nas_spec/changelog.md [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
 | Date       | Description of change                                                                                                 |
 | ---------- | --------------------------------------------------------------------------------------------------------------------- |
-| 2022/03/22 | Fixed invalid format for `authentication_date` |
+| 2022/03/22 | Added new scheme `cartes_bancaires` to enum `scheme` in Get and Create Sessions Responses                                                                           |
+| 2022/03/22 | Fixed invalid format for `authentication_date`                                                                        |
```
Comment (ben-ahmady-cko) [inline nas_spec/changelog.md:6]: Not completely sure how this got in but it looks fine

**Review by ben-ahmady-cko — APPROVED** [review]

**Pull request merged**